docs/wallets/cli/dcrwallet-tickets.md

```diff
@@ -150,17 +180,20 @@ A quick example:
 ```no-highlight
 enableticketbuyer=1
 ```
-
-If you are using a VSP, you should also add the following lines (all of these can be found on your VSP's "Tickets" page):
-
-```no-highlight
-ticketbuyer.votingaddress=<P2SH Address shared with VSP>
-pooladdress=<VSP's Fee Collection Address>
-poolfees=<VSP's Required Reward Fee>
-```
+You also need to specify options within your config file for your VSP as outlined [here.](#vsp-voting)
 
 If you don't want `ticketbuyer` to spend all of your funds, there is one more option which allows you to specify a balance which will not be spent:
 
 ```no-highlight
 ticketbuyer.balancetomaintainabsolute=<balance>
 ```
+Note: The info below is now considered "legacy" and will be phased out at a later date. It is recommended to use the process [here](#vsp-voting) to purchase tickets with a VSP as of version 1.6.0 and above as it better respects privacy and requires no sign-up with the VSP.
```

Comment:
Need an empty newline before this.

Without it, this text is formatted weirdly